README.md

```diff
@@ -4,110 +4,64 @@
 
-  While on Windows it's
-
-  ```
-  C:\> /path/to/new/virtual/environment/Scripts/activate
-  ```
+```toml
+  "emu-mps"
+```
 
-  Remember to replace `/path/to/new/virtual/environment` with the actual path to your virtual environment. Once the environment is activated, you can clone emu_mps and install it using
+to the list of `dependencies`.
 
-</details>
 
-### installing from the registry
+### Using `pip` or `pipx`
+To install the `pipy` package using `pip` or `pipx`
 
-When pip is configured to know about the pasqal registry, Emu-MPS installs as
+1. Create a `venv` if that's not done yet
 
-```bash
-pip install emu-mps
-```
-When pip is not already configured, the easiest way to do so, is to add a file `~/.config/pip/pip.conf` containing:
+```sh
+$ python -m venv venv
 
 ```
-[global]
-extra-index-url=https://gitlab.pasqal.com/api/v4/projects/597/packages/pypi/simple
-                possible.other.urls
-```
-
-As this shows, it is also possible to have multiple extra repositories configured. Note that the order is not important.
 
-It is also possible to add the `extra-index-url` to the `pip install` command directly, if you somehow don't want to create a `pip.conf` file.
+2. Enter the venv
 
-### installing from source
-git clone this [repository ](https://gitlab.pasqal.com/emulation/rydberg-atoms/emu-ct) or download
+If you're running Unix:
 
-
-Then, `cd` into the root folder of the repo and type
-
-```bash
-pip install -e .
+```sh
+$ . venv/bin/activate
 ```
 
-<details>
-  <summary>Guidelines for developers </summary>
-  We recommend using an environment, git clone the repository, then inside the `emu_mps` folder
+If you're running Windows:
 
-```bash
-pip install -e .
+```sh
+C:\> /path/to/new/virtual/environment/Scripts/activate
 ```
 
-  Also, the installation of pytest, nbmake, pre-commit.
+3. Install the package
 
-  Do not forget to run the unit test suite by simply running `pytest` command.
+```sh
+$ pip install emu-mps
+# or
+$ pipx install emu-mps
+```
 
-  Another way can be using hatch.
 
-  #### virtual environment with `hatch`
+Join us on [Slack](https://pasqalworkspace.slack.com/archives/C07MUV5K7EU) or by [e-mail](mailto:emulation@pasqal.com) to give us feedback about how you plan to use Emu-MPS or if you require specific feature-upgrades.
 
-  ```bash
-  python -m pip install hatch
-  python -m hatch -v shell
-  ```
+## Usage
 
-  When inside the shell with development dependencies, install first the pre-commit hook:
-  ```
-  pre-commit install
-  ```
-</details>
+For the time being, the easiest way to learn how to use this package is to look
+at the [examples](examples/emu_mps_examples).
 
-## Check the tutorial notebooks and example scripts
+See also the [full API documentation](https://pqs.pages.pasqal.com/emu_mps/).
 
 For more information, you can check the tutorials and examples located in the [examples folder](https://gitlab.pasqal.com/emulation/rydberg-atoms/emu-ct/-/tree/main/examples?ref_type=heads)
```

Comment:
this is essentially the same folder as the one you refer to in line 62. This entire sentence can be removed, I think.